Modules/Core/Common/src/itkFileOutputWindow.cxx

```diff
@@ -33,7 +33,6 @@ FileOutputWindow::FileOutputWindow()
 FileOutputWindow::~FileOutputWindow()
 {
   delete m_Stream;
-  m_Stream = nullptr;
```

Comment:
This sets the pointer to be null, to indicate that the pointer is no longer valid. It does not attempt to access the object which was pointed at. If I remember correctly, this is a well established pattern.

Reply:
@dzenanz It's an old discussion, actually... [Is it worth setting pointers to NULL in a destructor?](https://stackoverflow.com/questions/3060006/is-it-worth-setting-pointers-to-null-in-a-destructor)

When the destructor is finished, the FileOutputWindow object is gone, and so is its m_Stream data member 🤷

Reply:
But why are we not using a smart pointer? 🐱

Reply:
Thanks @blowekamp  A smart pointer would been much better. It's just because m_Stream isn't private (it's protected) that I did not do so immediately. If you're very strict, you _might_ say that changing a `protected` data member is an API change. (On the other hand, the _P_ in _API_ stands for _Public_, not _Protected_ 😸.)

Reply:
Doesn't API=Application Programming Interface?